lib/api/CFieldDataCategorizer.cc

```diff
@@ -159,12 +162,19 @@ int CFieldDataCategorizer::computeCategory(const TStrStrUMap& dataRowFields) {
         return -1;
     }
 
-    bool exampleAdded = m_ExamplesCollector.add(categoryId, fieldValue);
-    bool searchTermsChanged = this->createReverseSearch(categoryId);
+    bool exampleAdded{m_DataCategorizer->addExample(categoryId, fieldValue)};
```

Comment:
Just realised that while we have a comment in `STYLEGUIDE.md` for (non-)boolean accessor functions:

```Boolean accessor functions MUST be named as isComplete NOT complete```

we don't seem to have anything similar for mutator functions. Should there be? If so what?

Reply:
In the old coding standards Word document the section titled "Non-boolean accessor functions MUST be named as 'clientId' NOT 'getClientId'" had a detailed description of "Accessor functions MUST NOT be prefixed with get or set."

I think people are still adhering to setter methods not being prefixed with set.

But this is not a setter method, it's adding an item to a list.  So I think it's better that it does contain a verb to indicate that it's not a setter method.  If it was just `example` then people might assume it was a setter method setting a single example.